ansible/group_vars/omero-hosts.yml

```diff
@@ -242,7 +242,7 @@ omero_web_config_set:
 omero_web_apps_packages:
 - omero-mapr==0.5.2
 - omero-iviewer==0.14.0
-- idr-gallery==3.11.0
+- idr-gallery==3.12.0
```

Comment:
```
ERROR: Could not find a version that satisfies the requirement idr-gallery==3.12.0 (from versions: 3.5.0, 3.6.0, 3.6.1, 3.6.2, 3.7.0, 3.7.1, 3.8.0, 3.9.0, 3.9.1, 3.9.2, 3.10.0, 3.10.1, 3.11.0)
```

Looks like the publishing workflow on https://github.com/IDR/idr-gallery/ needs to be fixed like in https://github.com/ome/omero-gallery/pull/112